### DISCUSSION TITLE
unit conversion

### DISCUSSION THREAD

**Discussion by lezeski** [Blair2004/NexoPOS#1328] [Ideas]
If I bought 1 tray of eggs, that is 30 eggs. My expectation is if I sell 1 egg, the system should be able to convert the tray into 30 eggs and therefore allow me to sell. After selling 1 egg, the stock quantity should read as 29 eggs and 0 trays.

![CAP1](https://user-images.githubusercontent.com/122467997/213978316-a5780341-2315-4cf3-892e-f66fadadee1e.PNG)
![CAP2](https://user-images.githubusercontent.com/122467997/213978313-dde0e6ed-33e2-45f8-953a-35f969cee262.PNG)
![CAP3](https://user-images.githubusercontent.com/122467997/213978311-149613e5-3a56-42b9-83b9-ccf91dee4746.PNG)
![CAP4](https://user-images.githubusercontent.com/122467997/213978300-5a92b24f-9e98-40ab-9963-cb07dc31e6f6.png)

**Comment by Blair2004** [comment]
Approved suggestion :)
